keylime.conf

```diff
@@ -258,6 +258,16 @@ revocation_notifier_port = 8992
 # size of the actual payloads
 max_upload_size = 104857600
 
+# Severity labels for revocation events strictly ordered from least severity to
+# highest severtiy.
+severity_labels = ["info", "notice", "warning", "err", "crti", "alert", "emerg"]
+
+# Severity policy that matches different event_ids to the severity label.
+# The rules are evaluated from the beginning of the list and the first match is
+# used. The event_id can also be a regex.
+severity_policy = [{"event_id": "ima.*", "severity_label": "alert"}, \
```

Comment:
Is this just a sample severity_policy or the default/recommended one?

Reply:
Just a sample one for testing. I'll provide one that maps all events to the highest severity in the final PR.